proto/proto/penumbra/view/v1alpha1/view.proto

```diff
@@ -268,3 +285,13 @@ message SpendableNoteRecord {
   // The source of the note (a tx hash or otherwise)
   core.chain.v1alpha1.NoteSource source = 8;
 }
+
+message SwapRecord {
+  core.crypto.v1alpha1.StateCommitment swap_commitment = 1;
+  core.dex.v1alpha1.SwapPlaintext swap = 2;
+  uint64 position = 3;
+  core.crypto.v1alpha1.Nullifier nullifier = 4;
+  core.dex.v1alpha1.BatchSwapOutputData output_data = 5;
+  optional uint64 height_claimed = 6;
+  core.chain.v1alpha1.NoteSource source = 7;
+}
```

Comment:
This is the analogue of the `SpendableNoteRecord`, with information on the `Swap` that's helpful for a client (e.g., it needs the position and output data to produce the claim).